File: snippets/contact-center/create-incoming_call-endpoint/create-incoming_call-endpoint.py
@app.route("/incoming_call", methods=['GET', 'POST'])
def incoming_call():
      resp = VoiceResponse()
   with resp.gather(num_digits="1", action="/enqueue_call", timeout=10) as g:
      g.say("Please select from the following departments", language="en")
      g.say("For sales press one, for support press two, for marketing press three", language="en")
   return Response(str(resp), mimetype='text/xml')

File: snippets/contact-center/create-incoming_call-endpoint/create-incoming_call-endpoint.cs
public ActionResult Incoming_call()
{   
    var action = new Uri("/Home/enqueue_call");
    var response = new VoiceResponse();
    var gather = new Gather(timeout: 3, numDigits: 1, action: action);
    gather.Say("Please select from the following departments");
    gather.Say("For sales press one, for support press two, for marketing press three", language: "en-gb");

    response.Append(gather);
           
    return Content(response.ToString(), contentType: "text/xml");
}
